Tasks/Beresten/Task1/task1.py

```diff
@@ -0,0 +1,28 @@
+
+
+
+Initial_Amount_On_Accoutn = int(input('Enter the amount you want to put on the account:'))
+P = Initial_Amount_On_Accoutn
+Annual_Deposit_Rate = int(2)
+I = Annual_Deposit_Rate
+Deposit_Period = int(input('Enter the number of years:'))
+t = Deposit_Period
+Amount_Of_The_Days = int(t * 365)
```

Comment:
it's a money-related task, so I advise to use the Decimal data type for the calculation, -1 point